inform7/Internal/Inter/Architecture32Kit/Sections/Input Output.i6t

```diff
@@ -319,6 +322,17 @@ to document all of that.
         glk_put_char_stream(gg_commandstr, 10); ! newline
     }
   .KPContinue;
+
+    uwl = UnicodeWhitespace + WORDSIZE;
+    for ( ix = 1 : ix <= (a_buffer-->0) : ix++ )
+      if (((a_buffer-->ix) <= 32) || ((((a_buffer-->ix) >= 8192) && ((a_buffer-->ix) <= 8202))))
```

Comment:
The binary search would almost certainly be faster than the checks for 8192-8202, but then you have to list them all in the array.